### PR TITLE
fix DedupWorker core for CI

### PR DESCRIPTION
**Pull request by TKaxe**
### Changelog category :
- New Feature
- Bug Fix
- Improvement
- Performance Improvement
- Backward Incompatible Change
- Build/Testing/Packaging Improvement
- Documentation (changelog entry is not required)
- Other
- Not for changelog (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

...

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

### CODE REVIEW

**Comment by TKaxe** [comment]
close #852

**Review by jenrryyou — COMMENTED** [review]
LGTM

**Review by jenrryyou — APPROVED** [review]

File: src/Core/BackgroundSchedulePool.h [hunk 1]
```diff
@@ -184,6 +184,8 @@ class BackgroundSchedulePoolTaskHolder
     BackgroundSchedulePoolTaskHolder & operator=(const BackgroundSchedulePoolTaskHolder & other) noexcept = delete;
     BackgroundSchedulePoolTaskHolder & operator=(BackgroundSchedulePoolTaskHolder && other) noexcept = default;
 
+    BackgroundSchedulePoolTaskInfoPtr hasTask() { return task_info; }
```
Comment (WangTaoTheTonic) [inline src/Core/BackgroundSchedulePool.h:187]: `hasXXX` usually returns bool type, can we refactor it?

Comment (dmthuc) [reply src/Core/BackgroundSchedulePool.h:187]: Why don't you use the bool conversion operator?

**Pull request merged**